config/environments/production.rb

```diff
@@ -142,6 +142,7 @@
   # Admin email to receive the 'new user needs verification' emails
   config.user_verification_recipients = (ENV["USER_VERIFICATION_RECIPIENTS"] || '').split(' ')
 
-  config.admin_email = ENV["ADMIN_EMAIL"]
-  config.cap_api_key = ENV["CAP_API_KEY"]
+  # config.admin_email = ENV["ADMIN_EMAIL"]
+  # config.cap_api_key = ENV["CAP_API_KEY"]
+  config.professor_verification_email = "bjohnson@law.harvard.edu" 
```

Comment:
I'd suggest setting these as environment variables. That way if someone leaves or you just want to change it, it isn't a whole redeploy process. Deploying can change overtime and doesn't always remain simple. It also removes changes like this from the commit history since it's more user data than app data.